scss/bundles/dx.carmine.compact.scss

```diff
@@ -0,0 +1,4 @@
+@use "../widgets/generic" with (
+$color: "carmine",
+$size: "compact"
```

Comment:
```suggestion
    $color: "carmine",
    $size: "compact"
```

Reply:
I have plans to manually refactor all scss after generator removing.